src/Factory/OrderFactoryInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\AdminOrderCreationPlugin\Factory;
+
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+
+interface OrderFactoryInterface extends FactoryInterface
```

Comment:
Do we have to extend `FactoryInterface`?

Reply:
We need, at least for the Behat (we use Sylius context) :/